jib-maven-plugin/src/main/java/com/google/cloud/tools/jib/maven/PluginConfigurationProcessor.java

```diff
@@ -59,6 +68,35 @@ static AbsoluteUnixPath getAppRootChecked(JibPluginConfiguration jibPluginConfig
     }
   }
 
+  /**
+   * Gets the value of the {@code <to><image>} parameter. If this parameter is null,
```

Comment:
s/`<to>`/`<from>`/ along with similar rewording recommendation.

Reply:
Done